src/screens/MainApp/DiagnoseRequest/screens/Gallery/styles.js

```diff
@@ -0,0 +1,33 @@
+import { StyleSheet } from 'react-native'
+import { scale } from 'react-native-size-matters/extend'
+import { verticalScale } from 'react-native-size-matters'
+
+export const SCREEN_WIDTH = scale(327)
+
+const styles = StyleSheet.create({
+  listContainer: {
+    marginTop: verticalScale(26),
+    padding: scale(12),
+    paddingBottom: verticalScale(130)
+  },
+  button: {
+    position: 'absolute',
+    bottom: verticalScale(26)
+  },
+  box: {
+    margin: scale(4),
+    borderRadius: scale(6),
+    backgroundColor: '#5ECE841A',
```

Comment:
Wow i did not know this was a thing (`#rrggbbaa`), consider creating a `theme.colors.transparentPrimary` or `theme.colors.primaryAlpha`